src/backends/zora-graph/ZoraGraphDataSource.ts

```diff
@@ -93,15 +99,17 @@ export class ZoraGraphDataSource implements ZoraGraphDataInterface {
     return object;
   }
 
-  loadNFT = async (tokenContract: string, tokenId: string) => {
-    if (tokenContract.toLowerCase() != this.mediaContractAddress.toLowerCase()) {
+  loadNFT = async ({ contract, id }: { contract: string; id: string }) => {
+    if (contract.toLowerCase() != this.mediaContractAddress.toLowerCase()) {
       throw new Error('Not zora media contract');
     }
-    return await this.nftGraphDataLoader.load(tokenId);
+    return await this.nftGraphDataLoader.load(id);
   };
 
-  async loadNFTs(tokenContractAndId: readonly string[]) {
-    return await this.nftGraphDataLoader.loadMany(tokenContractAndId);
+  async loadNFTs(nfts: readonly NFTIdentifier[]) {
+    return await this.nftGraphDataLoader.loadMany(
+      nfts.map(({ contract, id }) => getAddress(`${contract}:${id}`))
```

Comment:
this should be:
`${getAddress(contract)}${SEPERATOR}${id}` getAddress will break if you pass it `:${id}` it just needs the contract address.

Reply:
+